### PR TITLE
HSC-37: Add Initial Encounter forms

### PR DESCRIPTION
**Pull request by reagan-meant**

### CODE REVIEW

**Comment by rbuisson** [comment]
Mistakenly closed.

File: configuration/concepts/vitals.csv [hunk 1]
```diff
@@ -1,19 +1,10 @@
...
 c642aeb3-95d3-4d1b-aa5b-1273b438d14f,,CIEL:5092,Oxygen Saturation,SaO2,,Saturation d'oxygène,SaO2,,Question,Numeric,,,,,%,95,100,0,100,,
 06188def-b659-4feb-8899-d513f49b67fb,,,Blood Pressure Set,Blood Pressure,,Groupe tension artérielle,Tension artérielle,,ConvSet,N/A,,,Tension artérielle systolique; Tension artérielle diastolique,,,,,,,,
-f074942b-666b-4ca3-9f0e-1103128b3ae2,,CIEL:1343,Mid-Upper Arm Circumference,Mid-Upper Arm Circumference,,Périmètre bracchial,Périmètre bracchial,,Question,Numeric,,,,TRUE,cm,,,,,,
-773d36e9-119e-452c-91ee-4ac90245e504,,CIEL:1114,Vital Signs,Vital Signs,,Signes vitaux,Signes vitaux,,ConvSet,N/A,,,Pouls; Température; Frequence respiratoire; Saturation d'oxygène; Groupe tension artérielle,,,,,,,,
\ No newline at end of file
+3fe6460f-3d37-463e-9fb9-df42ab15aec8,,,Pain Scale,Pain Scale,,Échelle de Douleur,Échelle de Douleur,,Question,Numeric,,,,FALSE,,,,0,10,,
+773d36e9-119e-452c-91ee-4ac90245e504,,,Vital Signs,Vital Signs,,Signes Vitaux,Signes Vitaux,,ConvSet,N/A,,,Échelle de Douleur;Pouls;Température;Frequence respiratoire;Saturation d'oxygène;Groupe tension artérielle,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/vitals.csv]: "Échelle de douleur"

File: configuration/concepts/reviewofsystems.csv [hunk 2]
```diff
@@ -0,0 +1,75 @@
+Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Short name:en,Short name:fr,Description:en,Description:fr,Data class,Data type,Complex data handler,Answers,Members,Allow decimals,Units,Normal low,Normal high,Absolute low,Absolute high,_version:1,_order:1600
+e904d309-fb5c-452c-b9b1-8daf9a3eff15,,,Loss of Appetite,Perte d'appétit,Loss of Appetite,Perte d'appétit,,,Misc,N/A,,,,,,,,,,,
+4a9edbef-9ad9-40a3-a5d3-83a1674bc4df,,,General,Général,General,Général,,,Misc,Coded,,Perte d'appétit;Fièvre;Fatigue (R53);Sueurs nocturnes (R61.1);Autre,,,,,,,,,
+aa314e8c-aaf2-4ae6-80c8-a9b228852f8f,,,General Set,Ensemble général,General Set,Ensemble général,,,ConvSet,N/A,,,Général;Autres observations,,,,,,,,
+1a384669-a1fe-46bc-a7cb-4a1ed0556918,,,Discolorations,Décolorations,Discolorations,Décolorations,,,Misc,N/A,,,,,,,,,,,
+bbcc605d-ee86-42eb-8a04-73648cbc9ef3,,,Lesions,Lésions,Lesions,Lésions,,,Misc,N/A,,,,,,,,,,,
+8b53b799-9127-4a38-89bb-3f3d90405557,,,Tattoo,Tattoo,Tattoo,Tattoo,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: "Tatouage"

File: configuration/concepts/reviewofsystems.csv [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+db2ac1d4-6d8f-4c21-95a7-6bfd49faf42c,,,Skin,Peau,Skin,Peau,,,Misc,Coded,,Éruptions cutanées;Prurit dermatologique;Décolorations;Lésions;Tattoo;Autre,,,,,,,,,
+da0abb70-c1d3-4478-ac11-b03ebcc0689f,,,Skin Set,Ensemble de peau,Skin Set,Ensemble de peau,,,ConvSet,N/A,,,Peau;Autres observations,,,,,,,,
+d5d18115-8cf3-4b10-9110-32dec69db4f6,,,Pain,Douleur,Pain,Douleur,,,Misc,N/A,,,,,,,,,,,
+53521df8-e9f3-4e42-8649-7fe11df6bab2,,,Conjuctivitis,Conjonctivite,Conjuctivitis,Conjonctivite,,,Misc,N/A,,,,,,,,,,,
+26688603-992f-4171-a1d6-a5d344cdef29,,,Eye itching,Prurit yeux,Eye itching,Prurit yeux,,,Misc,N/A,,,,,,,,,,,
+71e58402-8b2e-4fe4-91bd-e2a220c92dc0,,,Teary eyes,Larmoiement,Teary eyes,Larmoiement,,,Misc,N/A,,,,,,,,,,,
+19a92519-e350-434e-ba92-1cc42d031821,,,Pallor,Pâleur,Pallor,Pâleur,,,Misc,N/A,,,,,,,,,,,
+784cdb78-3b6d-447b-9eee-e29b6e4feae7,,,Deaf,Surdité,Deaf,Surdité,,,Misc,N/A,,,,,,,,,,,
+e72e92ae-ca88-4a01-862b-ce3e16c25f96,,,Bleeding,Saignements,Bleeding,Saignements,,,Misc,N/A,,,,,,,,,,,
+03e3c003-767a-4bcd-88cc-932ed65dbab6,,,Nose Discharge,Nez Sécrétions,Nose Discharge,Nez Sécrétions,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: "Sécrétions nasales"

File: configuration/concepts/reviewofsystems.csv [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+aa314e8c-aaf2-4ae6-80c8-a9b228852f8f,,,General Set,Ensemble général,General Set,Ensemble général,,,ConvSet,N/A,,,Général;Autres observations,,,,,,,,
+1a384669-a1fe-46bc-a7cb-4a1ed0556918,,,Discolorations,Décolorations,Discolorations,Décolorations,,,Misc,N/A,,,,,,,,,,,
+bbcc605d-ee86-42eb-8a04-73648cbc9ef3,,,Lesions,Lésions,Lesions,Lésions,,,Misc,N/A,,,,,,,,,,,
+8b53b799-9127-4a38-89bb-3f3d90405557,,,Tattoo,Tattoo,Tattoo,Tattoo,,,Misc,N/A,,,,,,,,,,,
+db2ac1d4-6d8f-4c21-95a7-6bfd49faf42c,,,Skin,Peau,Skin,Peau,,,Misc,Coded,,Éruptions cutanées;Prurit dermatologique;Décolorations;Lésions;Tattoo;Autre,,,,,,,,,
+da0abb70-c1d3-4478-ac11-b03ebcc0689f,,,Skin Set,Ensemble de peau,Skin Set,Ensemble de peau,,,ConvSet,N/A,,,Peau;Autres observations,,,,,,,,
+d5d18115-8cf3-4b10-9110-32dec69db4f6,,,Pain,Douleur,Pain,Douleur,,,Misc,N/A,,,,,,,,,,,
+53521df8-e9f3-4e42-8649-7fe11df6bab2,,,Conjuctivitis,Conjonctivite,Conjuctivitis,Conjonctivite,,,Misc,N/A,,,,,,,,,,,
+26688603-992f-4171-a1d6-a5d344cdef29,,,Eye itching,Prurit yeux,Eye itching,Prurit yeux,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Prurit oculaire

File: configuration/concepts/reviewofsystems.csv [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+03e3c003-767a-4bcd-88cc-932ed65dbab6,,,Nose Discharge,Nez Sécrétions,Nose Discharge,Nez Sécrétions,,,Misc,N/A,,,,,,,,,,,
+1607769b-1096-4399-8c22-981a8b4742e8,,,Sinus Congestion,Congestions des sinus,Sinus Congestion,Congestions des sinus,,,Misc,N/A,,,,,,,,,,,
+8137f84b-ca74-487a-ac3e-4c3364e97716,,,Missing,Manquante,Missing,Manquante,,,Misc,N/A,,,,,,,,,,,
+87a835f5-46c7-4e81-bfd4-687fa62d1cf1,,,Loose,Branlante,Loose,Branlante,,,Misc,N/A,,,,,,,,,,,
+2de43a2e-e82c-4e04-a276-17dd4db40c54,,,Teeth,Dents,Teeth,Dents,,,Misc,Coded,,Manquante;Branlante,,,,,,,,,
+fd55467a-6453-4fc1-a9a7-0ff78e9baa4e,,,Tongue,Langue,Tongue,Langue,,,Misc,N/A,,,,,,,,,,,
+08887d3e-78fe-4dff-84e4-3eea4287df02,,,Pallet,Palais,Pallet,Palais,,,Misc,N/A,,,,,,,,,,,
+bcda6677-244d-466c-9c33-98c02855b990,,,Mouth and Throat Deformation,Déformation de la bouche et de la gorge,Mouth and Throat Deformation,Déformation de la bouche et de la gorge,,,Misc,Coded,,Langue;Palais,,,,,,,,,
+55887656-fb44-4fcc-a9bc-4d73101ed31c,,,Mouth and Throat,Bouche et de la gorge,Mouth and Throat,Bouche et de la gorge,,,Misc,Coded,,Douleur;Dysphagie (R13);Autre,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Bouche et gorge

File: configuration/concepts/reviewofsystems.csv [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+1a705716-d86c-427f-9bb4-6487fe748988,,,Mouth and Throat Set,Bouche et de la gorge set,Mouth and Throat Set,Bouche et de la gorge set,,,ConvSet,N/A,,,Dents;Déformation de la bouche et de la gorge;Bouche et de la gorge;Autres observations,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Groupe bouche et gorge

File: configuration/concepts/reviewofsystems.csv [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+c1eef44a-42ae-41a1-a78f-26a354c59334,,,Eyes,Yeux,Eyes,Yeux,,,Misc,Coded,,Trouble visuel;Douleur;Conjonctivite;Prurit yeux;Écoulement des yeux;Larmoiement;Pâleur;Autre,,,,,,,,,
+bb9bf90d-1fad-4240-a79d-dbd6fcc1e038,,,Eyes Set,Ensemble d'yeux,Eyes Set,Ensemble d'yeux,,,ConvSet,N/A,,,Yeux;Autres observations,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Groupe yeux

File: configuration/concepts/reviewofsystems.csv [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+4463d07e-fa84-4f31-8bb1-afc1d04a4501,,,Ears,Oreilles,Ears,Oreilles,,,Misc,Coded,,Acouphènes;Vertiges;Surdité;Écoulement;Douleur;Autre,,,,,,,,,
+85bd41e4-5982-4ffe-b0c3-92fcc6c7476b,,,Ears Set,Ensemble d'oreilles,Ears Set,Ensemble d'oreilles,,,ConvSet,N/A,,,Oreilles;Autres observations,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Groupe oreilles

File: configuration/concepts/reviewofsystems.csv [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+669eb2ed-6fe3-47d0-b9ed-17e3b8af1e85,,,Nose,Nez,Nose,Nez,,,Misc,Coded,,Saignements;Nez Sécrétions;Congestions des sinus;Autre,,,,,,,,,
+b970d95f-ebc7-4fcb-b163-cd8be32328e3,,,Nose Set,Ensemble de nez,Nose Set,Ensemble de nez,,,ConvSet,N/A,,,Nez;Autres observations,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: let's replace "ensemble/set" by "groupe" everywhere.

File: configuration/concepts/reviewofsystems.csv [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+97de94dd-7fc6-414b-afbe-934d6489e10e,,,Orthopnea,Orthopnée,Orthopnea,Orthopnée,,,Misc,N/A,,,,,,,,,,,
+063cb6f4-f54d-4ee6-878b-6c683cacac8a,,,Platypnea,Platypnée,Platypnea,Platypnée,,,Misc,N/A,,,,,,,,,,,
+7629a17f-9fc1-4fb6-a766-059cc88620a3,,,Productive Cough,Expectoration,Productive Cough,Expectoration,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: "Toux productive"

File: configuration/concepts/reviewofsystems.csv [hunk 11]
```diff
@@ -0,0 +1,75 @@
...
+78d7751f-5ff3-4cef-b9a7-4c81d55fa5ed,,,Respiratory,Respiratoire,Respiratory,Respiratoire,,,Misc,Coded,,Toux (R05);Hemoptysie (R04.2);Dyspnee (R06.0);Orthopnée;Platypnée;Expectoration;Autre,,,,,,,,,
+97c382db-646b-4ee8-8bc4-fb630a9fc1e6,,,Productive Cough Details,Détails de la toux productive,Productive Cough Details,Détails de la toux productive,,,Misc,Text,,,,,,,,,,,
+1885d33d-9384-4b41-a2a4-4550fea7bf2d,,,Respiratory Set,Respiratoire set,Respiratory Set,Respiratoire set,,,ConvSet,N/A,,,Respiratoire;Détails de la toux productive;Autres observations,,,,,,,,
+00e0aad9-7fa0-48af-9000-003bfd616875,,,Paroxysmal Nocturnal Dyspnea,Dyspnée paroxystique Nocturne,Paroxysmal Nocturnal Dyspnea,Dyspnée paroxystique Nocturne,,,Misc,N/A,,,,,,,,,,,
+25ec3417-f125-4e8c-8862-7f50123b4f81,,,Dyspnea with Exertion,Dyspnée d'Effort,Dyspnea with Exertion,Dyspnée d'Effort,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Dyspnée d'effort

Comment (rbuisson) [reply configuration/concepts/reviewofsystems.csv]: Actually I'm not sure that the right translation. Here the french name suggests the dyspnea is happening on phyiscal effort. Which I don't think "with Exertion" means.

File: configuration/concepts/reviewofsystems.csv [hunk 12]
```diff
@@ -0,0 +1,75 @@
...
+2059f006-1c3f-4ab4-8d05-7c6d51c662d5,,,Cardiovascular Set,Ensemble cardiovasculaire,Cardiovascular Set,Ensemble cardiovasculaire,,,ConvSet,N/A,,,Cardiovasculaire;Autres observations,,,,,,,,
+1b5792c6-9fb0-445f-a588-823b8b5d091d,,,Melena,Méléna,Melena,Méléna,,,Misc,N/A,,,,,,,,,,,
+45b8d250-4f1a-47c2-ab1e-41205d41d675,,,Gastrointestinal,Gastrointestinal,Gastrointestinal,Gastrointestinal,,,Misc,Coded,,Dysphagie (R13);Douleurs abdominales;La nausée;Vomissement (R11);Hematemese (K92.0);Méléna;Diarrhee (K52.9);Constipation;Hématochézie;Dyschésique;Autre,,,,,,,,,
+660f5c64-d23c-4b98-a9fd-b507834fcab0,,,Gastrointestinal Set,Gastrointestinal set,Gastrointestinal Set,Gastrointestinal set,,,ConvSet,N/A,,,Gastrointestinal;Autres observations,,,,,,,,
+6788f174-9be5-40af-b938-379cf3745c88,,,Head,Tête,Head,Tête,,,Misc,Coded,,Cephalee (R51);Vertiges;Masse;Convulsion (R56.8);Autre,,,,,,,,,
+212f798e-7625-459f-8512-a88606408126,,,Head Set,Tête set,Head Set,Tête set,,,ConvSet,N/A,,,Tête;Autres observations,,,,,,,,
+4305753b-937e-4a22-b299-e1680af09919,,,Urgency,Impériosité,Urgency,Impériosité,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Urgenturie (if this means sudden need to urinate)

File: configuration/concepts/reviewofsystems.csv [hunk 13]
```diff
@@ -0,0 +1,75 @@
...
+58978ab4-9b80-4121-a2b9-b9202b2021c2,,,Hair Problem,Problème de cheveux,Hair Problem,Problème de cheveux,,,Misc,N/A,,,,,,,,,,,
+f607ea8e-ef4c-4a0b-9d61-d7cd51c921c1,,,Heat Intolerance,Tolérance à la chaleur,Heat Intolerance,Tolérance à la chaleur,,,Misc,N/A,,,,,,,,,,,
+b1eee37f-bf5a-4608-a108-7c603a3ec94e,,,Endocrine,Endocrinien,Endocrine,Endocrinien,,,Misc,Coded,,Polydipsie;Polyurie;Peau séche;Tolérance à la chaleur;Problème de cheveux;Autre,,,,,,,,,
+adf2279f-c34f-4db6-a8e1-21f0486ca490,,,Endocrine Set,Endocrinien set,Endocrine Set,Endocrinien set,,,ConvSet,N/A,,,Endocrinien;Autres observations,,,,,,,,
+5ebdd207-22e6-4bbd-bda1-940bb6ecf2e1,,,Joint Pain,Douleur Articulaire,Joint Pain,Douleur Articulaire,,,Misc,N/A,,,,,,,,,,,
+9a611f07-07f5-4aaf-b1b0-cfcf42b53d66,,,Joint Deformity,Déformation articulaire,Joint Deformity,Déformation articulaire,,,Misc,N/A,,,,,,,,,,,
+bed907b6-56f8-4bd5-8961-01a18e0cbfe2,,,Musculoskeletal,Musculosquelettique,Musculoskeletal,Musculosquelettique,,,Misc,Coded,,Douleur Articulaire;Gonflement des articulations;Myalgie;Raideur articulaire;Déformation articulaire;Autre,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Musculo-squelettique

File: configuration/concepts/reviewofsystems.csv [hunk 14]
```diff
@@ -0,0 +1,75 @@
...
+b43a3268-4f2f-4eb8-83e8-2ed9e07ad107,,,Body Part,Partie du corps,Body Part,Partie du corps,,,Misc,Text,,,,,,,,,,,
+c37bb128-d23e-41f0-b7b6-6186b98637ac,,,Musculoskeletal Set,Musculosquelettique set,Musculoskeletal Set,Musculosquelettique set,,,ConvSet,N/A,,,Musculosquelettique;Partie du corps;Autres observations,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Groupe musculo-squelettique

File: configuration/concepts/reviewofsystems.csv [hunk 15]
```diff
@@ -0,0 +1,75 @@
...
+2fe1f75c-61bc-4573-88ca-3d1b18daf9ec,,,Bruises,Écchymoses,Bruises,Écchymoses,,,Misc,N/A,,,,,,,,,,,
+94cc4a0a-281b-4c04-ae3a-cbaf60189aad,,,Skin and Lymphatic,Peau et lymphatique,Skin and Lymphatic,Peau et lymphatique,,,Misc,Coded,,Écchymoses;Lésions;Lymphadenopathie;Autre,,,,,,,,,
+2494b6d5-c4a1-4b44-a389-191a874e639c,,,Skin and Lymphatic Set,Peau et lymphatique set,Skin and Lymphatic Set,Peau et lymphatique set,,,ConvSet,N/A,,,Peau et lymphatique;Partie du corps;Autres observations,,,,,,,,
+40fb91c5-3a0b-4e96-b67c-f3da8f7d8900,,,Memory Issues,Troubles mémoires,Memory Issues,Troubles mémoires,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Troubles de la mémoire

File: configuration/concepts/reviewofsystems.csv [hunk 16]
```diff
@@ -0,0 +1,75 @@
...
+039cb6b6-ad9f-4abc-9abd-c208ec75cbbb,,,Neurosis,Neurose,Neurosis,Neurose,,,Misc,N/A,,,,,,,,,,,
+c9ee4047-e2ec-4f99-b179-762493627fd1,,,Motor Skills Disorder,Trouble de la motricité,Motor Skills Disorder,Trouble de la motricité,,,Misc,N/A,,,,,,,,,,,
+12596595-2f20-4786-ba73-ae8bcd046f33,,,Language Disorder,Trouble du langage,Language Disorder,Trouble du langage,,,Misc,N/A,,,,,,,,,,,
+01aa32c4-7878-45ba-b2d6-7cb3b49e5332,,,Self Care Skill Deficiency,Déficit de compétences en soins personnels,Self Care Skill Deficiency,Déficit de compétences en soins personnels,,,Misc,N/A,,,,,,,,,,,
+4f8d3b98-bd77-4586-b851-2232fb76090c,,,Neuropsychiatric and Psychomotor,Neuro psychiatrique and psycho moteur,Neuropsychiatric and Psychomotor,Neuro psychiatrique and psycho moteur,,,Misc,Coded,,Perte de mémoire;Depression (F32.9);Psychose (F29);Anxiete (F41.9);Neurose;Trouble de la motricité;Trouble du langage;Déficit de compétences en soins personnels;Autre,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/reviewofsystems.csv]: Neuro-psychiatrique et psycho-moteur

**Review by rbuisson — CHANGES_REQUESTED** [review]
Pretty much only reviewed "reviewofsystems.csv". Will do the rest later.

Btw, why no "_" this time?

File: configuration/concepts/commons.csv [hunk 17]
```diff
@@ -139,4 +139,9 @@ c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Visit number,Numéro de la visite,Visit n
 c4b48a7a-7c13-4e7d-89ad-1df68c5c6801,,,First Visit,Première visite,First Visit,Première visite,,,Misc,Coded,,bf885a9d-9f98-4990-8c20-ac4ace816bfe;2120763c-6c49-4133-b532-1e931ab4858a,,,,,,,,,
 f5e7648d-9dfb-4506-9602-000ab940a197,,,Quantity Dispensed,Quantité distribuée,Quantity Dispensed,Quantité distribuée,,,Misc,Numeric,,,,TRUE,,,,,,,
 9d20f065-9568-4769-80b5-f596907a8805,,,Details,Précisions,Details,Précisions,,,Misc,Text,,,,,,,,,,,
-2da93612-9727-4a29-ae6d-518fc62265ae,,,All Observation Templates,All Observation Templates,,,,,ConvSet,N/A,,,,,,,,,,,
\ No newline at end of file
+2da93612-9727-4a29-ae6d-518fc62265ae,,,All Observation Templates,All Observation Templates,,,,,ConvSet,N/A,,,,,,,,,,,
+7a5b55b6-4617-42b9-808c-82116a259a6e,,,Minibus,Tap Tap,Minibus,Tap Tap,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Was this "Tap tap" in english or in french originally?
Anyway it should probably stay "Tap tap" in all languages.

File: configuration/concepts/commons.csv [hunk 18]
```diff
@@ -139,4 +139,9 @@ c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Visit number,Numéro de la visite,Visit n
...
+8e84e437-35fa-4d31-8be9-9aa999f1ff94,,,Ambulated,Ambulant,Ambulated,Ambulant,,,Misc,N/A,,,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Ambulance

File: configuration/concepts/commons.csv [hunk 19]
```diff
@@ -139,4 +139,9 @@ c4b48a7a-7c13-4e7d-89ad-1df68c5c6802,,,Visit number,Numéro de la visite,Visit n
...
+0b290aeb-b4f6-4b37-b114-e7ce2e4c304c,,,Car,Voiture,Car,Voiture,,,Misc,N/A,,,,,,,,,,,
+f83eeb93-703a-4dd5-bd95-51a6580b4065,,,Mode of Arrival,Mode d'Arrivée,Mode of Arrival,Mode d'Arrivée,,,Misc,Coded,,Tap Tap;Ambulant;Voiture;Autre,,,,,,,,,
```
Comment (rbuisson) [inline configuration/concepts/commons.csv]: Moyen de transport

**Review by rbuisson — CHANGES_REQUESTED** [review]
Reviewed "commons.csv"

File: configuration/concepts/vitals.csv (hunk 1 above)
Comment (reagan-meant) [reply configuration/concepts/vitals.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 2 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 3 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 4 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 5 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 6 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 7 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 8 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 9 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 10 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 11 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: With exertion implies when the person does any physical work, or movement

File: configuration/concepts/reviewofsystems.csv (hunk 12 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 13 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 14 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 15 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/reviewofsystems.csv (hunk 16 above)
Comment (reagan-meant) [reply configuration/concepts/reviewofsystems.csv]: Done

File: configuration/concepts/commons.csv (hunk 17 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 18 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

File: configuration/concepts/commons.csv (hunk 19 above)
Comment (reagan-meant) [reply configuration/concepts/commons.csv]: Done

**Pull request merged**